### PR TITLE
Updating 4 inscriptions

### PR DESCRIPTION
**Pull request by Jamesjhua**
ISic000471, ISic000415, ISic000399, ISic000395

### CODE REVIEW

**Comment by Jamesjhua** [comment]
Adding an additional 5 inscriptions from 30 April ; ISic 000370, 000375, 000377, 000380, 000478

File: inscriptions/ISic000478.xml [hunk 1]
```diff
@@ -190,14 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#JH">
+                <ab>
+                    <lb n="1"/><name>Hercolei</name>
+                    <lb n="2"/><name>Tesorus</name>
```
Comment (JonPrag) [inline inscriptions/ISic000478.xml:201]: thesaurus (i.e. treasure), not a name (so leave as is, but remove name tag)

File: inscriptions/ISic000478.xml [hunk 2]
```diff
@@ -190,14 +195,20 @@
...
+            <div type="apparatus" resp="#JH">
+                <listApp>
+                    <app><note>Text from CIL</note></app>
+                    <app loc="line 2"><note>Tesorus : Gualt. ed. 1 ; Thesorus: Gualt. ed. 2</note></app>
+                </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#JH">
+                <p>Tesorus (dedicates this) to Hercules</p>
```
Comment (JonPrag) [inline inscriptions/ISic000478.xml:211]: 'treasure of hercules'

**Review by JonPrag — CHANGES_REQUESTED** [review]
Good work on this, thank you - various awkward things, some of which only presented today

**Comment by Jamesjhua** [comment]
Made corrections and 3 new inscriptions (2 Greek)

**Review by JonPrag — APPROVED** [review]
Thanks James; I've directly amended the last two Greek texts - important not to rely on the PHI texts, which are mostly rather garbled due to the conversion processes out of the original betacode, and to use one or more of the published texts as your direct basis.

**Comment by Jamesjhua** [comment]
Dear Jonathan, 

Thanks for letting me know - I’ll use the IG original text for the next ones. On a side note, would I be allowed to join just for the first hour (9-10am) for this Friday, please? I have a tutorial at 10am, but I can move it till later if needed.

Thanks.
All the best,
James

**Pull request merged**